pkg/controller/nodeipam/ipam/cloud_cidr_allocator.go

```diff
@@ -160,6 +160,11 @@ func NewCloudCIDRAllocator(client clientset.Interface, cloud cloudprovider.Inter
 				klog.Errorf("Received unexpected object: %v", originalObj)
 				return
 			}
+			if !meta.IsStatusConditionTrue(nw.Status.Conditions, string(networkv1.NetworkConditionStatusReady)) {
+				// ignore non-Ready Networks
+				klog.V(5).Infof("Ignoring non-Ready Network (%s) create event", nw.Name)
+				return
+			}
```

Comment:
a network is always in non-ready state when it is being created right? Why are we ignoring it? let me know if the flow is wrong:

1. CCM updates the NI annotation with this network
2. anetd reads it, and includes it in network-status
3. CCM then updates the networks ann

Reply:
we are ignoring it because we will skip if Network is not Ready. Should we just remove the whole AddFunc part ?